components/multiselect/multiselect_list.jsx

```diff
@@ -156,10 +160,25 @@ export default class MultiSelectList extends React.Component {
 
         const optionControls = options.map((o, i) => renderer(o, this.state.selected === i, this.props.onAdd));
 
+        if (this.props.ariaLabelRenderer) {
+            renderer = this.props.ariaLabelRenderer;
+        } else {
+            renderer = this.defaultAriaLabelRenderer;
+        }
+        const selectedOption = options.filter((o, i) => this.state.selected === i);
+        const ariaLabel = renderer(selectedOption[0] || {});
```

Comment:
Instead of passing an empty object into the renderer, would it make more sense to just set the label to be empty if we have no selected option?